lib/rspec/bash/matchers/called_with_arguments.rb

```diff
@@ -16,4 +17,21 @@
 
     called_with_correct_args && called_correct_number_of_times
   end
+
+  failure_message do |actual_command|
+    command_name = actual_command.command
+
+    formatted_expected_call = "#{command_name} #{expected_argument_list.join(' ')}"
+    formatted_actual_calls = actual_command.call_log.call_log_arguments.map do |arg_array|
+      "#{command_name} #{arg_array.join(' ')}"
+    end.join("\n")
```

Comment:
I'll play around with it, and at least see if we can make it so the matcher doesn't have to reach into the StubbedCommand to the CallLog, which is pretty smelly.